_includes/comparison.html

```diff
@@ -236,53 +236,6 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://docs.cilium.io/en/latest/installation/clustermesh/clustermesh/#gs-clustermesh" target="_blank">yes</a></td>
       </tr>
     </tbody>
-    <thead style="background-color: white; color: #ff4d67">
```

Comment:
I guess this was changed due to the transition from SMI to Gateway API ? @craigbox

Reply:
Yes; there's no point in listing compatibility with dead APIs.  I haven't proposed a section for compatibility with Gateway API/GAMMA APIs as (a) they're not done yet (b) they're tracked [here](https://gateway-api.sigs.k8s.io/implementations/).

Reply:
Thank you for your additions and your support :) I will merge your contributions.